fabric-network/src/impl/query/defaultqueryhandlerstrategies.js

```diff
@@ -1,40 +1,43 @@
 /**
- * Copyright 2018 IBM All Rights Reserved.
+ * Copyright 2018, 2019 IBM All Rights Reserved.
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 'use strict';
 
-const FabricConstants = require('fabric-client/lib/Constants');
-
 const SingleQueryHandler = require('fabric-network/lib/impl/query/singlequeryhandler');
 const RoundRobinQueryHandler = require('fabric-network/lib/impl/query/roundrobinqueryhandler');
 
 function getOrganizationPeers(network) {
-	return network.getChannel().getPeersForOrg().filter(hasChaincodeQueryRole);
+	return network.channel.getEndorsers(network.mspid);
```

Comment:
do you mean non low level calls, as the endorser list is public ?

Reply:
This has always been a point where users had to drop down to the low level API, at least to get the set of peers they are interested in. I guess it will stay that way for now. So long as there is a (hopefully easy) way for them to grab the peers (or even just names of peers) for the whole network and a given org (probably theirs) that doesn't require them to use undocumented / non-public internals

Reply:
I like the idea of adding `network.getPeers(<mspid>)` and `network.getOrderers(<mspid>)`